spec/graphql/upgrader/member_spec.rb

```diff
@@ -213,10 +213,87 @@ class UserInterface < BaseInterface
        end
       }
       new = %{
-       field :name, String, field: SomeField, null: false do
+       field :name, String, field: SomeField, null: true do
        end
       }
       assert_equal upgrade(old), new
     end
   end
+
+  describe 'complicated inline definition' do
+    # it 'upgrades without breaking syntax' do
+    #   old = %{
+    #     connection :example_connection, -> { ExampleConnectionType } do
+    #       argument :order_by, (GraphQL::InputObjectType.define do
```

Comment:
@esjee are you fine extracting this spec into a separate PR? It's pretty challenging to build a regex that matches inline types. We can ship this PR and take our time to fix this specific case.

Reply:
Sure, works for me.